js/components/controlBar.js

```diff
@@ -236,8 +236,12 @@ var ControlBar = React.createClass({
   handleMultiAudioClick: function () {
     this.configureMenuAutofocus(CONSTANTS.MENU_OPTIONS.MULTI_AUDIO);
 
-    this.togglePopover(CONSTANTS.MENU_OPTIONS.MULTI_AUDIO);
-    this.closePopovers();
+    if (this.props.responsiveView === this.props.skinConfig.responsive.breakpoints.lg.id) {
```

Comment:
I think we need some null checks for this variable
this.props.skinConfig.responsive.breakpoints.lg.id
Props and skinConfig are guaranteed to exist but I don't know about the rest.

Reply:
So make it something like this:

if (this.props.skinConfig.resposive &&
this.props.skinConfig.responsive.breakpoints &&
this.props.skinConfig.responsive.breakpoints.lg && 
(this.props.responsiveView === this.props.skinConfig.responsive.breakpoints.lg.id) ){
//do stuff
}

Reply:
@jpfranco @aeng7  Correct me if I'm wrong.

Reply:
\+